openmessaging-api/src/main/java/io/openmessaging/extension/ExtensionClient.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.openmessaging.extension;
+
+/**
+ * <p>
+ * This interface contains some methods are used for getting configurations related implementation. but this interface
+ * are not mandatory.
+ * </p>
+ *
+ * @version OMS 1.0.0
+ * @since OMS 1.0.0
+ */
+public interface ExtensionClient {
```

Comment:
Is there any way to get a instance for the new added interface io.openmessaging.extension.ExtensionClient.

There should be a getExcensionCleint method in the MessagingAccessPoint or other interfaces.

Reply:
yep, I also think it should be a method which can be found in Producer or Consumer. what's your opinion?

Reply:
Aggree that.